vcx/libvcx/Cargo.toml

```diff
@@ -13,7 +13,7 @@ path = "src/lib.rs"
 crate-type = ["staticlib","rlib", "dylib"]
 
 [features]
-default = ["nullpay"]
+default = ["no_payments"]
```

Comment:
Is libnullpay no longer being used? Do we need to move to libno_payments? This will affect other builds as well.

Reply:
I was testing, as null pay was failing. will remove this.